rmw_connext_shared_cpp/src/shared_functions.cpp

```diff
@@ -430,6 +438,14 @@ create_node(const char * implementation_identifier, const char * name, size_t do
   }
   memcpy(const_cast<char *>(node_handle->name), name, strlen(name) + 1);
 
+  node_handle->namespace_ =
+    reinterpret_cast<const char *>(rmw_allocate(sizeof(char) * strlen(namespace_) + 1));
+  if (!node_handle->namespace_) {
+    RMW_SET_ERROR_MSG("failed to allocate memory for node namespace");
+    goto fail;
+  }
+  memcpy(const_cast<char *>(node_handle->namespace_), namespace_, strlen(namespace_) + 1);
```

Comment:
this is link to the previous comment, where I believe that `memcpy` is the right approach here and `strdup` isn't.